modules/cpmso-yaml-provider-spec-vsphere.adoc

```diff
@@ -1,22 +1,53 @@
 // Module included in the following assemblies:
 //
-// * machine_management/control-plane-machine-management.adoc
+// * machine_management/cpmso-configuration.adoc
 
 :_content-type: REFERENCE
 [id="cpmso-yaml-provider-spec-vsphere_{context}"]
 = Sample vSphere provider specification
 
-When you create a control plane machine set for an existing cluster, the provider specification must match the `providerSpec` configuration in the control plane `machine` CR that is created by the installation program. You can omit any field that is set in the failure domain section of the CR.
-
-In the following example, `<cluster_id>` is the infrastructure ID that is based on the cluster ID that you set when you provisioned the cluster. If you have the OpenShift CLI installed, you can obtain the infrastructure ID by running the following command:
-
-[source,terminal]
-----
-$ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
-----
+When you create a control plane machine set for an existing cluster, the provider specification must match the `providerSpec` configuration in the control plane `machine` CR that is created by the installation program. 
 
 .Sample vSphere `providerSpec` values
 [source,yaml]
 ----
-
+providerSpec:
+  value:
+  apiVersion: machine.openshift.io/v1beta1
+  credentialsSecret:
+    name: vsphere-cloud-credentials <1>
+  diskGiB: 120 <2>
+  kind: VSphereMachineProviderSpec <3>
+  memoryMiB: 16384 <4>
+  metadata:
+    creationTimestamp: null
+  network: <5>
+    devices:
+    - networkName: <vm_network_name>
+  numCPUs: 4 <6>
+  numCoresPerSocket: 4 <7>
+  snapshot: ""
+  template: <vm_template_name> <8>
+  userDataSecret:
+    name: master-user-data <9>
+  workspace: 
+    datacenter: <vcenter_datacenter_name> <10>
+    datastore: <vcenter_datastore_name> <11>
+    folder: <path_to_vcenter_vm_folder> <12>
+    resourcePool: <vsphere_resource_pool> <13>
+    server: <vcenter_server_ip> <14>
 ----
+<1> Specifies the secret name for the cluster. Do not change this value.
+<2> Specifies the VM size for the control plane machines.
+<3> Specifies the cloud provider platform type. Do not change this value. 
+<4> Specifies the memory allocated for the control plane machines.
+<5> Specifies the network on which the control plane is deployed.
+<6> Specifies the number of CPUs allocated for the control plane machines.
+<7> Specifies the number of cores for the control plane machine CPUs.
```

Comment:
cores per control plane machine CPU?

Wondering if the current wording is ambiguous as to whether this is a total or per CPU